src/StreamStats.jl

```diff
@@ -1,15 +1,15 @@
 module StreamStats
     import StatsBase, Distributions
+    import Base.mean, Base.std, Base.var, Base.cov, Base.cor
+    import Base.maximum, Base.minimum
 
-    export update!, state, nobs, rand, ci
+    export update!, state, nobs, rand, ci, sample
```

Comment:
@johnmyleswhite , @StefanKarpinski , any thoughts on what to do with the `ci` method?  Why is it problematic if both packages implement a method call `ci`?  Will they clobber one another in some way?  It seems funny to have to use the name of somebody else's package if your package isn't calling it.